l3kernel/expl3.dtx

```diff
@@ -254,7 +254,7 @@
 % \begin{quote}
 %   \meta{module} and \meta{description}
 % \end{quote}
-% these both give information about the command.
+% that both give information about the command.
```

Comment:
I think this is arguable - 'these' seems OK to me

Reply:
Either is OK. But maybe clearer:

and these two

Reply:
> and these two both give . . .

@car222222 I like this suggestion! It's applied.